test/unit/sth_test.js

```diff
@@ -969,6 +969,110 @@ describe('Data removal', function() {
   );
 });
 
+describe('POST /admin/log', function() {
```

Comment:
I tend to think as you do, that tests may be slightly more verbose than normal code, as one of the main features that is expected from a test its readability. All that said, I would have chosen to refactor the tests to a function in the same test file, just over the tests themselves, or in a different test file (I think splitting tests in multiple files help a lot with this readability problems).  I leave that for future refactor of the tests (as I think there is a STH refactor pending).